build/index.html

```diff
@@ -1,15 +1,218 @@
+      </picture>
+    </header>
+
+    <main class="page-main">
+      <h1 class="visually-hidden">Круизы в Антарктикую. Волшебный мир айсбергов и пингвинов</h1>
+      <section class="main-header">
+        <h2 class="visually-hidden">Приветствующая секция</h2>
+        <div class="main-header__wrapper">
+          <dl class="main-header__list">
+            <dt class="main-header__title">Круизы в Антарктику</dt>
+            <dd class="main-header__description">Волшебный мир айсбергов и пингвинов</dd>
+          </dl>
+        </div>
+      </section>
+
+      <section class="main-description">
+        <h2 class="visually-hidden">Секция с описанием путешествия</h2>
+        <div class="main-description__wrapper">
+          <dl class="main-description__list">
+            <dt class="main-description__title">Путешествие на край Земли</dt>
+            <dd class="main-description__text">Это волшебное путешествие поможет вам познакомиться поближе с удивительным миром Антарктики. Наш круизный лайнер отвезет вас на край Земли, где вы сможете увидеть: ледяные айсберги, несколько видов пингвинов, китов, морских котиков и многое другое.</dd>
+          </dl>
+          <picture>
+            <source type="image/webp" media="(min-width: 1366px)" srcset="img/penguins-desktop@1x.webp, img/penguins-desktop@2x.webp 2x">
+            <source type="image/webp" media="(min-width: 768px)" srcset="img/penguins-tablet@1x.webp, img/penguins-tablet@2x.webp 2x">
+            <source type="image/webp" srcset="img/penguins-mobile@1x.webp, img/penguins-mobile@2x.webp 2x">
+    
+            <source media="(min-width: 1366px)" srcset="img/penguins-desktop@1x.jpg, img/penguins-desktop@2x.jpg 2x">
+            <source media="(min-width: 768px)" srcset="img/penguins-tablet@1x.jpg, img/penguins-tablet@2x.jpg 2x">
+            <img src="img/penguins-mobile@1x.jpg" srcset="img/penguins-mobile@2x.jpg" alt="Пингвинчики">
+          </picture>
+        </div>
+      </section>
+
+      <section class="main-advantages">
+        <h2 class="visually-hidden">Секция с преимуществами</h2>
+        <div class="main-advantages__wrapper">
+          <p class="main-advantages__title">Преимущества</p>
+          <div class="main-advantages__container">
+            <!-- Вставить еще две карточки после утверждения формата -->
+            <div class="main-advantages__card">
+              <svg class="main-advantages__logo" width="27" height="32">
+                <use href="img/sprite_auto.svg#logo-ship"></use>
+              </svg>
+              <dl class="main-advantages__list">
+                <dt class="main-advantages__title">Небольшой круизный лайнер</dt>
+                <dd class="main-advantages__description">Круизный лайнер, рассчитанный всего лишь на 300 гостей, поможет избежать шума и очередей и даст возможность насладиться видами и природой.</dd>
+              </dl>
```

Comment:
* декоративные иконки скрыть от Скрин ридера
* опять непонятный dl
```suggestion
              <svg class="main-advantages__logo" width="27" height="32" aria-hidden="true">
                <use href="img/sprite_auto.svg#logo-ship"></use>
              </svg>
              <h3 class="main-advantages__title">Небольшой круизный лайнер</h3>
              <p class="main-advantages__description">Круизный лайнер, рассчитанный всего лишь на 300 гостей, поможет избежать шума и очередей и даст возможность насладиться видами и природой.</p>
```